app/src/components/features/mocksV2/MockEditorIndex/Editor/index.tsx

```diff
@@ -336,6 +336,7 @@ const MockEditor: React.FC<Props> = ({
             height={220}
             value={headersString}
             defaultValue={headersString}
+            defaultPrettified={false}
```

Comment:
No need to pass false here as this prop is already assigned false as default value in the editor component.